charts/ratify/templates/assign.yaml

```diff
@@ -0,0 +1,50 @@
+apiVersion: mutations.gatekeeper.sh/v1beta1
+kind: Assign
+metadata:
+  name: mutate-pod-image
+spec:
+  match:
+    scope: Namespaced
+    kinds:
+    - apiGroups: ["*"]
+      kinds: ["Pod"]
+    excludedNamespaces:
+      - "gatekeeper-system"
+      {{- if ne .Release.Namespace "gatekeeper-system"}}
+      - {{ .Release.Namespace }}
+      {{- end }}
+  applyTo:
+  - groups: [""]
+    kinds: ["Pod"]
+    versions: ["v1"]
+  location: "spec.containers[name:*].image"
+  parameters:
+    assign:
+      externalData:
+        provider: ratify-mutation-provider
+
+---
+apiVersion: mutations.gatekeeper.sh/v1beta1
+kind: Assign
+metadata:
+  name: mutate-deployment-image
+spec:
+  match:
+    scope: Namespaced
+    kinds:
+    - apiGroups: ["apps"]
+      kinds: ["Deployment"]
```

Comment:
Why only pods and deployment? What about jobs, cronjobs, statefulsets, daemonsets and replicasets (xref: https://kubernetes.io/docs/concepts/workloads/controllers/)?

Reply:
@aramase Good point. Will add those. It seems like you can only provide a static `location`. Some of the workload resources like `CronJobs` have a slightly different location path. Is the guidance to create new `Assign` resources per unique location path?

Reply:
@sozercan thoughts on this?

Reply:
I've added it for now. We can follow up if guidance changes.